src/Button/SimpleButton/SimpleButton.spec.jsx

```diff
@@ -9,12 +8,12 @@ import { SimpleButton } from '../../index';
 
 describe('<SimpleButton />', () => {
   it('is defined', () => {
-    expect(SimpleButton).not.to.be(undefined);
+    expect(SimpleButton).not.toBeUndefined();
```

Comment:
Has Jest its own assertion library or is another library than `expect.js` used?

Reply:
Jest has its own assertion syntax -> see PR description.